src/tests/ftest/osa/osa_offline_extend.py

```diff
@@ -87,16 +87,7 @@ def run_offline_extend_test(self, num_pool, data=False):
                                                   rank, scm_size,
                                                   nvme_size)
             self.log.info(output)
-
-            pver_extend = self.get_pool_version()
```

Comment:
If you remove this `pver_extend`, doesn't it cause problem in line 103 and 105 where you access it?

Reply:
The logic is different. Checking the pool version caused unwanted "Rebuild State"... Sometimes pool version is updated before the rebuild is done. The logic is changed such that the pool version is ready only after rebuild is complete. It is done only once. This seems to make the script reliable.